tests/unit_tests/test_stats.py

```diff
@@ -296,6 +296,12 @@ def test_mixture_clip():
     mix_same = mix.clip(1e-6, inplace=True)
     assert mix_same is mix
 
+    # Make sure clip removes low probability distributions
+    d_small = openmc.stats.Uniform(0., 1.)
+    d_large = openmc.stats.Uniform(2., 5.)
+    mix = openmc.stats.Mixture([1e-10, 1.0], [d_small, d_large])
+    mix_clip = mix.clip(1e-3)
+    assert mix_clip.distribution == [d_large]
```

Comment:
only question here is if `inplace=False` by default should we expect to get back the same `Univariate` objects in the resulting clipped `Mixture`? My initial reaction was that I would expect the underlying distributions to be different objects. I don't think any users will really care about this, but it could have side-effects if someone then modified `d_large` after mixing, not expecting it to effect the `Mixture` distribution, because there is a specific `inplace` kwarg to that effect in `clip`.

Reply:
That's an interesting thought. We could default to copying any distribution that is not a discrete distribution when `inplace=False`, although that could be a bit wasteful from a memory perspective. I don't have a strong feeling either way since I don't think this will be a common issue, so let me know which way you prefer.

Reply:
me neither we can leave as is